README.md

```diff
@@ -137,12 +134,13 @@ Contributions, issues, and feature requests are welcome!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- Thanks to Microverse for this opportunity
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](LICENSE.md) licensed.
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ Optional ] I think it would be better if you add a `LICENSE` to the root directory of your project and then update the `link` to the License on your readme file. This will make it possible for someone who would like to use your code to know exactly how the project is Licensed.